install/update_0905_91.php

```diff
@@ -522,16 +589,21 @@ function update0905to91() {
             foreach ($items_to_update as $templates_id => $type) {
                if (isset($type['itemtype'])) {
                   if (isset($type['items_id'])) {
-                     $DB->queryOrDie("DELETE FROM `$table`
-                                      WHERE `num` = '".$item_num."'
-                                            AND `tickettemplates_id` = '".$templates_id."'",
-                                     "Associated items migration : delete $table itemtypes");
+                     $DB->deleteOrDie($table, [
+                           'num'                => $item_num,
+                           'tickettemplates_id' => $templates_id,
+                        ],
+                        "Associated items migration : delete $table itemtypes"
+                     );
                   }
-                  $DB->queryOrDie("UPDATE `$table`
-                                   SET `num` = '".$item_num."'
-                                   WHERE `num` = '".$itemtype_num."'
-                                         AND `tickettemplates_id` = '".$templates_id."'",
-                                 "Associated items migration : delete $table itemtypes");
+                  $DB->updateOrDie($table, [
+                        'num' => $item_num
+                     ], [
+                        'num'                => $itemtype_num,
+                        'tickettemplates_id' => $templates_id,
+                     ],
+                     "Associated items migration : delete $table itemtypes"
```

Comment:
Not linked to your changes,but should be fixed:
```suggestion
                     "Associated items migration : update $table itemtypes"
```